server/utils/prober.js

```diff
@@ -182,8 +182,8 @@ function parseTags(format, verbose) {
     file_tag_comment: tryGrabTags(format, 'comment', 'comm', 'com'),
     file_tag_description: tryGrabTags(format, 'description', 'desc'),
     file_tag_genre: tryGrabTags(format, 'genre', 'tcon', 'tco'),
-    file_tag_series: tryGrabTag(format, 'series'),
-    file_tag_seriespart: tryGrabTag(format, 'series-part'),
+    file_tag_series: tryGrabTag(format, 'series', 'show'),
```

Comment:
We need to use the method `tryGrabTags` here to support passing in multiple tags. They should all be using `tryGrabTags` actually and the `tryGrabTag` method isn't necessary. That is just a remnant of when only 1 tag was accepted in for each mapping.